assets/css/_logged_in_as.scss

```diff
@@ -0,0 +1,25 @@
+.c-logged-in-as__container {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.c-logged-in-as__icon {
+  height: 32px;
+  width: 32px;
+}
+
+.c-logged-in-as__text {
+  font-weight: 400;
+  font-size: 14px;
+  color: #6e7175;
```

Comment:
As @lemald pointed out about a different color (that got `git`-ed away), this color may want to be replaced with `$color-gray-700`, although if we do that before [this PR](https://github.com/mbta/skate/pull/2365) is merged, then the color will be slightly wrong, since `$color-gray-700` is currently #3c3f4c

Reply:
```suggestion
  color: $color-gray-700;
```

Reply:
On further thought - the color is more than slightly wrong - the other links on this page are `gray-600`, not `gray-700`, so the text winds up looking pretty out of place. I think we should keep this as-is and follow up after the bootstrap colors PR is merged.

Reply:
Per Slack discussion: let's go with gray-700 for this PR since we're doing other work to integrate the new color palette changes.

Reply:
If I'm synthesizing correctly from all of the other discussions, I'm going to wait for https://github.com/mbta/skate/pull/2365 to get merged, and then use the new_v2 (or 2024) gray-700.

Reply:
Aaaand here we go: https://github.com/mbta/skate/pull/2383/commits/9c5ea5b2c3bc03fbf1381c07477966eb00e5421c